src/platform/forms-system/src/js/validation.js

```diff
@@ -84,7 +89,7 @@ export function transformErrors(errors, uiSchema) {
       error.argument,
     );
     if (newMessage) {
-      return _.set('message', newMessage, error);
+      return set('message', newMessage, error);
```

Comment:
:grimacing: I'm a bit concerned about removing `set` from this one. I looked up where this is being used and I think it would take too long to trace things down and be confident that nobody is relying on `transformErrors` to mutate their argument.

So it's fine to leave this one as-is.